core/src/main/java/org/elasticsearch/search/aggregations/support/AggregationPath.java

```diff
@@ -295,21 +290,13 @@ public void validate(Aggregator root) {
 
                 // we're in the middle of the path, so the aggregator can only be a single-bucket aggregator
 
-                if (!(aggregator instanceof SingleBucketAggregator)) {
+                if (!(aggregator instanceof SingleBucketAggregator) && !(aggregator instanceof AggregatorFactory.MultiBucketAggregatorWrapper)) {
```

Comment:
I just wonder: wouldn't it make sense to extract `first` out of `MultiBucketAggregatorWrapper` instead?